cluster/etcd-cluster.yaml

```diff
@@ -70,7 +70,7 @@ SenzaInfo:
       Description: VpcID to use for the security groups being setup.
   - EtcdImage:
       Description: Docker image for the etcd instances.
-      Default: "registry.opensource.zalan.do/acid/etcd-cluster:3.4.2-p23"
+      Default: "registry.opensource.zalan.do/teapot/etcd-cluster:3.4.2-master-4"
```

Comment:
Maybe we should update etcd to a fixed version: https://twitter.com/gyuhox/status/1263563998055690240?s=21

Reply:
Sure, but I want to finish the migration first and then test another image.